packages/manager/modules/pci/src/projects/project/private-networks/delete/translations/Messages_fr_FR.json

```diff
@@ -1,8 +1,8 @@
 {
-  "pci_projects_project_network_private_delete": "Suppression du réseau privé",
+  "pci_projects_project_network_private_delete": "Supprimer le réseau privé",
   "pci_projects_project_network_private_delete_confirm": "Confirmer",
   "pci_projects_project_network_private_delete_cancel": "Annuler",
-  "pci_projects_project_network_private_delete_confirmation": "Êtes-vous sûr de supprimer ce réseau privé : {{ name }} ?",
+  "pci_projects_project_network_private_delete_confirmation": "Souhaitez-vous réellement supprimer le réseau privé {{name}} ?",
```

Comment:
Is : needed in delete confirmation message ? 
Souhaitez-vous réellement supprimer le réseau privé  : {{ name }} ?

Reply:
This is how its there in the requirement doc, without `:`.